ZeldaFullEditor/Data/Overworld.cs

```diff
@@ -158,6 +158,8 @@ public class Overworld
 
         public OverlayAnimationData[] AllAnimationOverlays { get; set; } = new OverlayAnimationData[128];
 
+        public bool[] usedTiles16 = new bool[4096];
```

Comment:
num tile16 constant for this and the loop below

Reply:
Done